shared/elements/gaia_grid/js/grid_dragdrop.js

```diff
@@ -666,6 +672,12 @@
               return;
             }
 
+            if (this.gridView._launchingApp) {
```

Comment:
Actually, looking again, maybe launch/close are 100% reliable. I think I should restructure this around that...